### ISSUE TITLE
Event never received

### ISSUE THREAD

**Issue by GeoffreyPlitt** [segmentio/analytics-node#131]
I'm doing the simplest possible example, and I've waited several minutes-- segment.io is still showing no events ever received in the debugger.

```javascript
const Analytics = require('analytics-node')
const client = new Analytics('<write key from segment>')

client.track({
  event: 'event1',
  userId: 'user123'
})
```

I've configured Segment.io with a "Server" source, and I double-checked the write key is correct.

This seems like a "hello world" for this package and segment.io, not sure why it isn't working.

**Comment by f2prateek** [comment]
The library batches events, so when you call track it first buffers the message in memory and uploads to segment periodically.

It looks like in your example the program might be exiting before we ever try to flush. You can do so manually as well by calling the flush method. Alternatively you can use a smaller flush at value.

Also make sure you are using  the latest version  3.1.1 (3.1.0 had a regression which was fixed in 3.1.1).

**Comment by GeoffreyPlitt** [comment]
3.1.1 was the fix, thanks!
